.gitignore

```diff
@@ -58,3 +58,4 @@ libs/shared/src/containers/IDX/config.devlive.json
 apps/site-nft/src/translations
 apps/site-nft/lang
 
+environments/devlive.env
```

Comment:
Should I remove that?

Reply:
yes please.  Our devlive should contain no secrets (and anyone should be able to run it)